shiny/ui/_sidebar.py

```diff
@@ -107,17 +112,36 @@ def _from_string(cls, open: str) -> SidebarOpen:
         :
             A :class:`~shiny.ui.SidebarOpen` object.
         """
+        values_str = f"""'{"', '".join(cls._VALUES)}'"""
```

Comment:
Can `_values_str()` be turned into a `@staticmethod`? Then this variable `values_str` could be removed, and the same could be done in the `_as_open` method below.

Reply:
Ah, that's exactly what I wanted, thanks for the pointer!